src/client-app/src/models/game.ts

```diff
@@ -17,7 +17,7 @@ export class Game {
         this.board = convertFenToBoard(initialBoard);
         this.turn = (turn === 'w') ? 'white' : 'black';
         this.canCastle = processCastleState(castleState);
-        this.enPassantCoord = (enPassant === '-') ? null : null;
+        this.enPassantCoord = null;
```

Comment:
For this, we should be able to convert enPassant string to `Coord` if the enPassant string is not `'-'`.
Previously, I have just set this as null for both if else because we dont have the function to convert enPassant FEN to Coord.

Reply:
I deleted that assuming it only accepts default fen string and there will be no en passant targets from the start at all, and it is only in constructor. I actually wrote the code to convert Fen string to en passant coord, but i am not using it anywhere else so i deleted it.

If we are creating the game not only from default fen string and we are thinking of like reviewing some kind of board state and play from that stage and to implement that feature then, okay i will write back that code again.

Reply:
Yep, we are thinking to accept FEN string from the user as well so it's better to have that feature to convert enPassant string to Coord. Thanks!